public/robots.txt

```diff
@@ -1,3 +1,3 @@
 # https://www.robotstxt.org/robotstxt.html
 User-agent: *
-Disallow:
+Disallow: /
```

Comment:
Was there different robots.txt files provided somewhere for staging and production?

Reply:
LE UI is not on production yet so at this point I decided to disallow all pages. We have JIRA ticket https://helsinkisolutionoffice.atlassian.net/browse/LINK-393 to add sitemap to the robots.txt and have separate files for staging and production